toaster_msgs/srv/AddFactsToAgent.srv

```diff
@@ -1,4 +0,0 @@
-toaster_msgs/Fact[] facts
-string agentId
----
-bool answer
```

Comment:
Are you sure this service was used only in the db?
Otherwise it's good work :+1: